apisix/plugins/basic-auth.lua

```diff
@@ -39,7 +38,7 @@ local consumer_schema = {
     title = "work with consumer object",
     properties = {
         username = { type = "string" },
-        password = { type = "string" },
+        password = { type = "string", encrypted = true },
```

Comment:
We need to adjust the documentation of the corresponding plugin for this point.

Reply:
added